### DISCUSSION TITLE
October update didn't work as expected

### DISCUSSION THREAD

**Discussion by dajpes** [adonisjs/core#1860] [General]
Hi, after reading the October update I proceed to update my current adonis 5 project, but after doing it I got these errors:
![image](https://user-images.githubusercontent.com/11248816/98255603-90f64900-1f4b-11eb-84a6-086ed77f704d.png)

Here is my package.json

![image](https://user-images.githubusercontent.com/11248816/98255574-889e0e00-1f4b-11eb-866a-61e5c6f969f9.png)

**Comment by RomainLanz** [comment]
Hey @dajpes! 👋 

It seems that you haven't followed the steps to upgrade your application.
You need to remove `@adonisjs/fold` and `@adonisjs/ace` and also update all `@adonisjs` dependencies using the `@alpha` tag.

▶ https://preview.adonisjs.com/blog/october-2020-release#steps-to-upgrade

**Comment by dajpes** [comment]
After removing fold and ace, I got this error:

![image](https://user-images.githubusercontent.com/11248816/98280828-f9542300-1f69-11eb-8216-76e7c9f465be.png)

And this is my new package.json

![image](https://user-images.githubusercontent.com/11248816/98280858-04a74e80-1f6a-11eb-8144-26386be0a6b2.png)

**Reply by RomainLanz** [reply]
[In my previous answer](https://github.com/adonisjs/core/discussions/1860#discussioncomment-122369), I said that you should update **all dependencies**, you are still missing some, and don't forget the dev dependency `@adonisjs/assembler`.

**Reply by dajpes** [reply]
I'm trying to update @adonisjs/assembler but nothing happens:

![image](https://user-images.githubusercontent.com/11248816/98282383-29042a80-1f6c-11eb-9bdb-63f81674291c.png)

I don't why but the same happens with application and session dependency

**Reply by RomainLanz** [reply]
This is not how you update a dependency. You should run the following command.

```bash
$ npm i @adonisjs/core@alpha # and others...
$ npm i -D @adonisjs/assembler@alpha
```

Feel free to follow [our screencast](https://www.youtube.com/watch?v=6MBNw6JOaYQ) where we update a real AdonisJS application.

**Reply by dajpes** [reply]
Thank you Romain I did it as you say and now it works
